xAPI.md

```diff
@@ -675,6 +675,18 @@ LRS SHOULD* reject Statements containing such additional properties.
 
 <a name="statement-immutablity-and-exceptions" />
 
+Statements are evidence of a tracked learning experience.  Because this information is considered to be evidence, 
+the information represented in the Statement has already happened.  Thus, Statements will use the past tense of 
+Verbs when they display information and IRIs are encouraged to follow this same pattern of tense whenever possible.
```

Comment:
xAPI Verbs don't have tense, so you can't have "the past tense of Verbs". A replacement might be, "the natural language display for Verbs will usually be past tense, because the event represented in the Statement has already happened."